app/ante/ante_test.go

```diff
@@ -409,7 +409,18 @@ func (suite AnteTestSuite) TestAnteHandler() {
 				coinAmount := sdk.NewCoin(evmtypes.DefaultEVMDenom, sdk.NewInt(20))
 				amount := sdk.NewCoins(coinAmount)
 				gas := uint64(200000)
-				txBuilder := suite.CreateTestEIP712MsgEditValidator(from, privKey, "ethermint_9000-1", gas, amount)
+				txBuilder := suite.CreateTestEIP712MsgSubmitEvidence(from, privKey, "ethermint_9000-1", gas, amount)
+				return txBuilder.GetTx()
+			}, false, false, true,
+		},
+		{
+			"success- DeliverTx EIP712 submit proposal v1",
```

Comment:
are there any other `Any` types or messages affected by this issue? If so we should add them here

Reply:
Add `MsgExec`, these two should be the only Cosmos SDK Msgs with Array Any types. Ethermint does not contain any relevant msgs.